src/main/java/com/minecolonies/coremod/entity/ai/basic/AbstractEntityAIBasic.java

```diff
@@ -577,25 +548,32 @@ private IAIState lookForRequests()
                     );
                     return NEEDS_ITEM;
                 }
-                else
```

Comment:
This here is important, we should re-request if the pickup failed.

Reply:
shouldnt the normal AI handle that? Like if the request is completed and gets removed but it did not get the item it wanted, it naturally would run into re-requesting

Reply:
Not necessarily very timely. I think it's a neat speed improvement to have it here without any backlash.

Reply:
ok